cogs/quote_commands.py

```diff
@@ -91,8 +92,8 @@ async def quote(self, ctx:utils.Context, messages:commands.Greedy[discord.Messag
             if rows:
                 return await ctx.send(f"That message has already been quoted with quote ID `{rows[0]['quote_id']}`.")
             await db(
-                "INSERT INTO user_quotes (quote_id, guild_id, user_id, text, timestamp) VALUES ($1, $2, $3, $4, $5)",
-                quote_id, ctx.guild.id, user.id, text, timestamp
+                "INSERT INTO user_quotes (quote_id, guild_id, user_id, text, timestamp, message_link) VALUES ($1, $2, $3, $4, $5, $6)",
+                quote_id, ctx.guild.id, user.id, text, timestamp, link
```

Comment:
This only works with the addition of PR #11 - they should really have been in one PR but whatever.